docs/WIP/Tutorial_Motif_only_Matrix_Profile.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.        i = local_bsf[0]
</pre><p>This goes back to the point that I was making earlier. If you're going to split the indices apart here then you might as well keep them separate in the first place. Also, don't call it <code>local_bsf</code> if it was called <code>min_i</code> or <code>min_indices</code> before (hopefully, I'm not misunderstanding what the relationships are so please forgive me).  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/1046/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>